src/printer.js

```diff
@@ -575,49 +575,24 @@ function genericPrintNoParens(path, options, print, args) {
 
       return concat(parts);
     case "AwaitExpression":
-      parts.push("await");
-
-      if (n.all) {
-        parts.push("*");
-      }
-      if (n.argument) {
```

Comment:
```js
async function f() {
  await;
}
```

Chrome and babylon throw with unexpected `;`. Flow treats it as `await null;`.

cc @mroch, you may want to ban `await;` in flow.